app/controllers/houses_controller.rb

```diff
@@ -0,0 +1,55 @@
+class HousesController < ApplicationController
+  def index
+    @houses = current_user.houses
+  end
+
+  def new
+    @house = House.new
+  end
+
+  def edit
+    @house = House.find(params[:id])
+  end
+
+  def create
+    @house = House.new(user_params)
+    @house.user = current_user
+    @house.save
```

Comment:
You could have an error during save procedure. Usually is better to do like this:
```
if @house.save
  redirect_to houses_path
else
  flash.now[:error] = "Could not save the house"
  render action: "new"
end